init.lua

```diff
@@ -1,77 +1,302 @@
+local mod_name = "wielded_light"
```

Comment:
If mod.conf is set up properly, use `minetest.get_current_modname()`

Reply:
Done